hw_diag/tests/test_gcs_shipper.py

```diff
@@ -47,11 +47,11 @@ def test_upload_diagnostics_should_not_ship(self):
         retval = upload_diagnostics(diagnostics, False)
         self.assertIsNone(retval)
 
-    def test_diagnostics_to_gcs_payload(self):
+    def test_diagnostics_added_timestamp(self):
```

Comment:
Should prob be `test_add_timestamp_to_diagnostics`

Reply:
Edited: https://github.com/NebraLtd/hm-diag/blob/94bf73eec3ada1aeee700751fce0a00ba53ea390/hw_diag/tests/test_gcs_shipper.py#L50